properties/src/main/java/io/quarkus/qe/properties/providers/CustomConfigSource.java

```diff
@@ -12,7 +12,7 @@
 public class CustomConfigSource implements ConfigSource {
 
     private static final int ORDINAL = 999;
-    private static final String PROPERTIES_FILE = "/configsource.properties";
+    private static final String PROPERTIES_FILE = "/configsource.prop";
```

Comment:
This is caused by https://github.com/quarkus-qe/quarkus-test-framework/issues/169